### PR TITLE
Review pull request

### PR DESCRIPTION
**Pull request by thoreyjona**
# Pull request
## :heavy_exclamation_mark: Check list
- [X] Did you follow the TDD protocol?  _Irrelevant_
- [X] Did your code build?  
- [X] Did your code pass through Travis?  
- [X] Did you write a description about your branch and its purpose?
## :heavy_exclamation_mark: Description and purpose of the branch:

Adding checklist to review pull requests.

Signature

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/PowerRangersHug/TicTacToe/pull/6" target="_blank">ReviewNinja

**Comment by fanneykristjans** [comment]
hey im gonna merge this

File: docs/PullRequestDirections.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Pull request review checklist 
+
+- [ ] The code was built using TDD
```
Comment (reykjalin) [inline docs/PullRequestDirections.md:3]: This is cool !completed

**Review by fanneykristjans — APPROVED** [review]
Great! Well done.

**Pull request merged**